components/layout/AppMobileNavbar.vue

```diff
@@ -17,23 +17,26 @@
               $t('navbar.navbarLinkDefault')
             }}</NuxtLink>
           </li>
-          <li>
-            <NuxtLink class="font-bold block px-4 py-3" to="/projects?first">{{
+          <li class="font-bold block px-4 py-3">
+            <app-link to="/projects" store-name="projects-filter">{{
               $t('navbar.navbarProjects')
-            }}</NuxtLink>
+            }}</app-link>
           </li>
-          <li>
-            <NuxtLink
-              class="font-bold block px-4 py-3"
-              to="/dao-proposals?first"
-              >{{ $t('navbar.navbarDao') }}</NuxtLink
-            >
+          <li class="font-bold block px-4 py-3">
+            <app-link to="/dao-proposals" store-name="dao-proposals-filter">{{
+              $t('navbar.navbarDao')
+            }}</app-link>
+          </li>
+          <li class="font-bold block px-4 py-3">
+            <app-link to="/dao-voting" store-name="dao-voting-filter">{{
+              $t('leaderboard.meta.title')
+            }}</app-link>
           </li>
           <li>
-            <NuxtLink
-              class="font-bold block px-4 py-3"
-              to="/dao-voting?current"
-              >{{ $t('leaderboard.meta.title') }}</NuxtLink
+            <span
```

Comment:
question(blocking): Why is this one different than the others? Shouldn't this div also be an app-link?

Reply:
Fixed